bundles/core/org.eclipse.smarthome.core/OSGI-INF/metatype/org.eclipse.smarthome.core.net.NetUtil.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where does this file come from and what is the purpose of it? So far we do not have any metadata for services. Also we exclude `.xml` files within the `OSGI-INF` directory because they are auto generated by the DS-annotations.

Reply:
This file is used as metatype.  That way, the frequency (was cronexpression) can be manually configured in the UI.  Is that acceptable?